pep-0249.txt

```diff
@@ -1297,6 +1295,22 @@ Footnotes
     ``WHERE`` clause, or clearly document a different interpretation
     of the ``.rowcount`` attribute.
 
+.. [10] In Python 2 and earlier versions of this PEP, ``StandardError``
+    was used as the base class for all DB-API exceptions. Since
+    ``StandardError`` was removed in Python 3, database modules
+    targeting Python 3 should use ``Exception`` as base class instead.
+    Both ``StandardError`` and ``Exception`` are available as builtin
```

Comment:
`StandardError` was removed as mentioned in the previous sentence, it isn't a builtin :)

Reply:
I guess it's better to remove that sentence altogether. I mentioned it, since I had removed the import of the exceptions module (for Python 2).